superset-frontend/src/components/Table/Table.overview.mdx

```diff
@@ -183,14 +183,93 @@ The table displays a set number of rows at a time, the user navigates the table
 The default page size and page size options for the menu are configurable via the `pageSizeOptions` and `defaultPageSize` props.
 NOTE: Pagination controls will only display when the data for the table has more records than the default page size.
 
-<Story id="design-system-components-table-examples--many-columns" />
+<Story id="design-system-components-table-examples--pagination" />
 
 ```
 <Table pageSizeOptions={[5, 10, 15, 20, 25] defaultPageSize={10} />
 ```
 
 ---
 
+### Server Pagination
+
+The table be configured for async data fetching to get partial data sets while showng pagination controls that let the user navigate through data.
+To override the default pagin, which uses data.length to determine the record count, populate the `recordCount` prop with the total number for records
+contained in the dataset on the server that is being paged through. When the User navigates through the paged data it will invoke the `onChange` callback
+function enableing data fetching to occur when the user changes page.
```

Comment:
```suggestion
The table can be configured for async data fetching to get partial data sets while showing pagination controls that let the user navigate through data.
To override the default paging, which uses `data.length` to determine the record count, populate the `recordCount` prop with the total number of records contained in the dataset on the server being paged through. When the user navigates through the paged data it will invoke the `onChange` callback function enabling data fetching to occur when the user changes the page.
```